examples/validating-admission-webhook/loadbalancer_quota.rego

```diff
@@ -2,12 +2,14 @@
 package admission
 
 import data.k8s.matches
+
 deny[{
 	"id": "loadbalancer-service-quota",
 	"resource": {"kind": "services", "namespace": namespace, "name": name},
 	"resolution": {"message": "you cannot have more than 2 loadbalancer services in each namespace"},
 }] {
-    matches[["services", namespace, any_name, matched_services]]
-    count(matched_services) > 2
-    not re_match("^(openshift-*|kube-*)", namespace)
+    service := data.kubernetes.services[namespace][name]
```

Comment:
is this line needed? I don't see `service` used anywhere else.

Reply:
The line is needed to get reference to the namespace and name. the assignment to the service variable is not